src/DependencyInjection/Configuration.php

```diff
@@ -231,24 +231,33 @@ public function getConfigTreeBuilder()
                                             ->ifArray()
                                             ->then(static function ($items) {
                                                 foreach ($items as $key => $item) {
-                                                    if (\is_array($item)) {
-                                                        if (!\array_key_exists('label', $item) || !\array_key_exists('route', $item)) {
-                                                            throw new \InvalidArgumentException('Expected either parameters "route" and "label" for array items');
-                                                        }
-
-                                                        if (!\array_key_exists('route_params', $item)) {
-                                                            $items[$key]['route_params'] = [];
-                                                        }
-
-                                                        $items[$key]['admin'] = '';
-                                                    } else {
-                                                        $items[$key] = [
-                                                            'admin' => $item,
-                                                            'label' => '',
-                                                            'route' => '',
-                                                            'route_params' => [],
-                                                            'route_absolute' => false,
-                                                        ];
+                                                    if (!\is_array($item)) {
+                                                        $item = ['admin' => $item];
```

Comment:
What happens with the generated item if you dont include all the other default values?

Reply:
```
if (isset($item['admin']) && !empty($item['admin'])) {
            $admin = $this->pool->getInstance($item['admin']);

            $options = $admin->generateMenuUrl(
                'list',
                [],
                $item['route_absolute'] ? UrlGeneratorInterface::ABSOLUTE_URL : UrlGeneratorInterface::ABSOLUTE_PATH
            );
            $options['extras'] = [
                'label_catalogue' => $admin->getTranslationDomain(),
                'admin' => $admin,
            ];

            return $this->menuFactory->createItem($admin->getLabel(), $options);
        }
```

Reply:
Only `route_absolute` seems needed and missing.

Reply:
Seems it's before normalization the default value are added.

See the tests:
```
[
                'admin' => 'item1',
                'roles' => [],
                'route_params' => [],
                'route_absolute' => false,
            ]
```